content/landing_pages/PGsupport/Playground.md

```diff
@@ -30,7 +30,7 @@ You can write the code in JavaScript or Typescript. The playground software comp
 
 ## Overview
 
-![Playground Overview](/img/how_to/Introduction/playground.jpg)
+![Playground Overview](/public/img/how_to/Introduction/playground.jpg)

```

Comment:
You're right. It appears again to be a faulty preview on Github/me issue. Apologies again...

Reply:
Apparently only the webpage images remain genuinely are the broken links. However, I'm not sure what you mean by case, case of "Introduction". Should I close this PR? Feel free to share your honest opinion, I won't get be offended.

Reply:
You are not wasting my time at all :-)

We are ALWAYS happy there are people contributing to the framework. And that is being said with great honesty!

We also try not to discourage anyone from contributing again. I think this PR can be closed because you have solved issues that are not there in the live version (i.e. - github vs. the hosted version). But please feel free to continue submitting PRs you think are needed!

Reply:
Sounds good! Thanks a lot